pkg/handler/processor/guesser/format_guesser.go

```diff
@@ -37,7 +38,7 @@ var (
 
 func RegisterDocumentFormatGuesser(g DocumentFormatGuesser, name string) {
```

Comment:
same with this one. Should just return an error.

Reply:
Addressed ✅